package/MDAnalysis/analysis/__init__.py

```diff
@@ -59,7 +59,7 @@
 :mod:`~MDAnalysis.analysis.helanal`
     Analysis of helices with the HELANAL_ algorithm.
 
-:mod:`~MDAnalysis.analysis.hole`
+:mod:`~MDAnalysis.analysis.hole2.hole`
```

Comment:
Would be nicer to have a page for the hole2 package instead of going to its module. But can be separate PR if it requires work.

Reply:
So it turns out this `__init__` text isn't actually used anywhere for docs generation (as far as I can tell), the hole2 package links to: https://www.mdanalysis.org/docs/documentation_pages/analysis/hole2.html#hole-analysis-mdanalysis-analysis-hole2

I can just get rid of the `.hole` portion if you think it's more appropriate.

Reply:
I think hole2 is more appropriate. 

It looks as if the doc string requires an update anyway... probably getting rid of most of it and just saying something simple for anyone using `help(MDA.analysis)`. But can be different issue.

Reply:
I changed it in `fa48e06`, but on testing on a page for which docs actually do get built for, it seems like it won't link to anything. Either way if the whole thing is going to get re-written, and considering that these docs don't get built anyways, then it might be good enough for now?